javascript/javascript1/week2/StudentManager.js

```diff
@@ -0,0 +1,69 @@
+const class07Students = [];
+function addStudentToClass(studentName)
+ {
+    if(!studentName)
+    {
+        console.log('Please provide a student name');
+        return;
+    }
+    else if(typeof(studentName) != "string")
+    {
+        console.log('Please provide a proper string value');
+        return;
+    }
+else if(studentName==='Queen'|| class07Students.length<6)
+    {
+        class07Students.push(studentName);
+        console.log(studentName);
+        return;
+    }
+    else if(class07Students.includes(studentName)){
+        console.log(`Student ${studentName} is already in the class`);
+        return;
+       
+    }
+    else if(!studentName || studentName.length === 0)
```

Comment:
According to the coding principle DRY(Don't repeat yourself) this is an unecessary repetition as you have already checked for this up top